obstacle_adder.py

```diff
@@ -0,0 +1,118 @@
+import glob
+import cv2 as cv
+import numpy as np
+import utils
+from random import getrandbits
+
+
+img = None
+new_img = None
+mask = None
+new_mask = None
+obs = None
+obs_mask = None
+
+
+
+def obstacles_from_folder(folder, img_ext='.png'):
+    glob_selector = '{}/*{}'.format(folder, img_ext)
+    for file in glob.glob(glob_selector):
+        yield cv.imread(file)
+
+
+def load_obstacles(folder, img_ext='png'):
+    obstacles = []
+    obstacles_mask = []
+    for obs in obstacles_from_folder(folder):
+        mask = cv.cvtColor(obs, cv.COLOR_BGR2GRAY)
+        _, mask = cv.threshold(mask, 10, 255, cv.THRESH_BINARY)
+        obstacles.append(obs)
+        obstacles_mask.append(mask)
+    return np.asarray(obstacles), np.asarray(obstacles_mask)
+
+
+def __insert_image(image, mask, x, y, h, w):
+    global obs_mask, obs
```

Comment:
@Foxiy Rather than doing this globally, do you think it would make sense to just wrap this in an object? If that would lead to too much trouble do not even think about it.